src/Orc.Controls/Helpers/Win32Helper.cs

```diff
@@ -0,0 +1,83 @@
+﻿namespace Orc.Controls
+{
+    using System;
+    using System.Runtime.InteropServices;
+    using Orc.Controls.Win32;
+
+    // Credits:
+    // https://www.pinvoke.net/default.aspx/Enums/SetWindowPosFlags.html
+    public static class Win32Helper
```

Comment:
I think the win32helper should be different. Check Orchestra how we implemented this in a better manner.

Reply:
Not sure what's the best, in orchestra many of win32 extern methods are a part of classes. I've decided to create internal static User32 class instead of helper class and moved it to shared win32 folder with rect and enums.